src/n_body.rs

```diff
@@ -1,144 +1,224 @@
 // The Computer Language Benchmarks Game
 // http://benchmarksgame.alioth.debian.org/
 //
-// contributed by the Rust Project Developers
-// contributed by Matt Brubeck
+// contributed by the packed_simd developers
+// contributed by Andre Bogus
 // contributed by TeXitoi
 
-const PI: f64 = 3.141592653589793;
+use std::f64::consts::PI;
+use std::ops::*;
+
 const SOLAR_MASS: f64 = 4.0 * PI * PI;
-const YEAR: f64 = 365.24;
-const N_BODIES: usize = 5;
+const DAYS_PER_YEAR: f64 = 365.24;
+
+#[derive(Copy, Clone)]
+pub struct F64x2(f64, f64);
+
+#[derive(Copy, Clone)]
+pub struct F64x4(f64, f64, f64, f64);
+
+impl F64x2 {
+    pub fn splat(x: f64) -> F64x2 { F64x2(x, x) }
+    pub fn new(a: f64, b: f64) -> F64x2 { F64x2(a, b) }
+    pub fn sqrte(self) -> F64x2 { F64x2(self.0.sqrt(), self.1.sqrt()) }
```

Comment:
Why a final e at sqrte?

Reply:
That's for 'estimate'. I can remove it, as it should no longer be one.